### ISSUE TITLE
Fix govet error in httputil/buster_test.go (copy lock value)

### ISSUE THREAD

**Issue by dmitris** [golang/gddo#416]
Currently running `go tool vet .` on gddo code produces the following error in httputil/buster_test.go:
```
$ go tool vet .
buster_test.go:23: assignment copies lock value to cbs: httputil.CacheBusters contains sync.Mutex
```

would it be OK to change the offending line in https://opensource.git.corp.yahoo.com/golang/gddo/blob/master/httputil/buster_test.go#L23
```
cbs = CacheBusters{Handler: ss.FileHandler("buster_test.go")}
```
to `cbs.Handler = ss.FileHandler("buster_test.go")` ?  The tests run fine.

Another possibility would be to create a new variable ex. `cbs2` instead of reusing `cbs`, with the diff of two lines:

```diff
-       cbs = CacheBusters{Handler: ss.FileHandler("buster_test.go")}
+       cbs2 := CacheBusters{Handler: ss.FileHandler("buster_test.go")}

-       token = cbs.Get("/xxx")
+       token = cbs2.Get("/xxx")
```

`go tool vet` issues cause a problem for CI systems that fail the builds on vet errors (when building local versions of "gddo" using this codebase).

I'd be happy to submit a change, let me know if it wold be ok and if you have any preferences in terms of the options above.  Thanks.

**Comment by petedaltry** [comment]
Change the lines to 

	cbs := &CacheBusters{Handler: http.FileServer(http.Dir("."))}

and

	cbs = &CacheBusters{Handler: ss.FileHandler("buster_test.go")}

Note the addition of the address operator.

**Comment by dmitris** [comment]
sent https://go-review.googlesource.com/24260

**Comment by shurcooL** [comment]
Fixing a `go vet` issue is definitely a good idea and appreciated.

I've offered an alternative way to resolve it in a comment at https://go-review.googlesource.com/#/c/24260/1. (I made a typo there, it should be `token := cbs.Get("/xxx")` in the second block.)

**Comment by shurcooL** [comment]
This has been fixed in aaf246516d11966c40d171629ff6f8dd190e9e7c, so I'll close the issue.

Thanks for contributing!
